index.d.ts

```diff
@@ -0,0 +1,39 @@
+declare module 'athena-express' {
+    import * as aws from 'aws-sdk';
+    interface ConnectionConfigInterface {
+        aws: typeof aws;
+        s3: string;
+        getStats: boolean;
+    }
+
+    type GenericObject = {
+        [names: string]: any;
+    };
+
+    interface QueryResultsInterface<T> {
+        Items: T[];
+        DataScannedInMB: number;
+        QueryCostInUSD: number;
+        EngineExecutionTimeInMillis: number;
+        Count: number;
```

Comment:
should add `S3Location` here

Reply:
@darcoli added

Reply:
@darcoli also added `QueryExecutionId`, not quite sure if the typing of string is correct though.

Reply:
Yes typing is correct